components/layout/header.vue

```diff
@@ -69,23 +68,46 @@
             </div>
           </div>
           <div class="py-6 px-5 space-y-6">
-            <div class="grid grid-cols-2 gap-y-4 gap-x-8">
+            <div class="flex flex-col float-right space-y-4">
               <NuxtLink
                 v-for="item in navigation"
                 :key="item.name"
                 :to="item.to"
-                class="text-base font-medium text-gray-900 hover:text-gray-700"
+                class="text-bas font-light text-gray-900 hover:text-gray-700"
```

Comment:
Will need to change text color to #202020 according to prototype

Reply:
Okay,  now it's containing that color